test/desdemona/ui/table_test.cljs

```diff
@@ -1,8 +1,43 @@
 (ns desdemona.ui.table-test
   (:require
    [desdemona.ui.table :as table]
-   [cljs.test :as t :refer-macros [deftest testing is are]]))
+   [cljs.test :as t :refer-macros [deftest testing is are]]
+   [reagent.core :as r]
+   [reagent.session :as session]
+   [cljs.core.match :refer-macros [match]]))
+
+(def isClient (not (nil? (try (.-document js/window)
+                              (catch js/Object e nil)))))
+
+(def rflush r/flush)
```

Comment:
There wasn't much to test before when it comes to actual generated markup and its reactivity. I'll take a look at shrieker, thanks